src/spec.c

```diff
@@ -1094,6 +1172,8 @@ static void FieldSpec_RdbLoad(RedisModuleIO *rdb, FieldSpec *f, int encver) {
   }
 }
 
+#if 0
```

Comment:
?

Reply:
This is dead code, it's marked for removal.